pallets/subtensor/src/lib.rs

```diff
@@ -917,17 +919,22 @@ pub mod pallet {
     pub type AdjustmentAlpha<T: Config> =
         StorageMap<_, Identity, u16, u64, ValueQuery, DefaultAdjustmentAlpha<T>>;
 
-    #[pallet::storage] // --- MAP (netuid, who) --> (hash, weight) | Returns the hash and weight committed by an account for a given netuid.
+    #[pallet::storage] // --- DMAP ( netuid, who ) --> BTreeMap<nonce, (hash, block_number)> | Stores weight commitments for each account in a subnet.
     pub type WeightCommits<T: Config> = StorageDoubleMap<
         _,
         Twox64Concat,
-        u16,
+        u16, // netuid
         Twox64Concat,
         T::AccountId,
-        (H256, u64),
-        OptionQuery,
+        BTreeMap<u64, (H256, u64)>, // nonce -> (hash, block_number)
+        ValueQuery,
     >;
 
+    #[pallet::storage] // --- DMAP ( netuid, hotkey ) --> last processed nonce
+    #[pallet::getter(fn last_processed_nonce)]
```

Comment:
Getters are removed in Polkadot 1.11.0

Reply:
Yep, better practice to just refer to the storage directly.

Reply:
done